eZ/Bundle/EzPublishCoreBundle/Resources/config/fieldtype_services.yml

```diff
@@ -107,3 +107,5 @@ services:
             $locationService: '@ezpublish.api.service.location'
             $contentTypeService: '@ezpublish.api.service.content_type'
             $mappings: '$fieldtypes.ezimageasset.mappings$'
+
+    eZ\Publish\Core\FieldType\FieldTypeRegistry: ~
```

Comment:
AFAIR you don't need to repeat this definition if it's provided in `eZ/Publish/Core/settings/fieldtype_services.yml` and you don't want to override anything on the bundle level. It doesn't work the other way around. Or is it needed here anyway somehow?

Reply:
Looks like `eZ/Publish/Core/settings/fieldtype_services.yml` is being used in Integration tests and `eZ/Bundle/EzPublishCoreBundle/Resources/config/fieldtype_services.yml` is loaded to build app container. Removing entry from one file or another breaks either an application or tests.